base_country_state_translatable/models/res_country.py

```diff
@@ -0,0 +1,10 @@
+# © initOS GmbH 2017
```

Comment:
Can you replace the copyright sign with 'Copyright'. And also in the other files.

Reply:
@tarteo fixed